package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@mia-platform-internal/console-design-system-react",
   "packageManager": "yarn@4.2.2",
-  "version": "0.3.0",
```

Comment:
Why is the version bump in this pr?

Reply:
strangely, the npm version did not advance the version automatically. However, online we have 3.0.1.

Reply:
fake news it normally does 🤷